### DISCUSSION TITLE
is there any plan to support Telegraf to collect data of kvrocks

### DISCUSSION THREAD

**Discussion by luger1990** [apache/kvrocks#629] [Q&A]
we use tig stack to collect data . is there any plan to support Telegraf to collect data of kvrocks.

**Comment by git-hulk** [comment]
@luger1990 Thanks for your question.

We have no plan to do that, but Kvrocks can use https://github.com/KvrocksLabs/kvrocks_exporter to export the prometheus metric now. If you want to support the Telegraf collector, we're very happy to see this happens.
